resources/views/backend/routes/create.blade.php

```diff
@@ -168,41 +81,146 @@
    @parent
-   <script>
-       $(function () {
-         $('div.alert').not('.alert-danger').delay(5000).fadeOut(350);
-         $('.select2').select2();
-         $('#add_item').on('click',function(e){
-        e.preventDefault();
-        //get selected option
-        var uniqueid = Date.now();
-        var routeid = $('#item-selector').find(":selected").data('startitemid');
-        var endid = $('#item-selector2').find(":selected").data('enditemid');
-        var startname = $('#item-selector').find(":selected").data('startitemname');
-        var endname = $('#item-selector2').find(":selected").data('enditemname');
-        var markup = "<tr item-id='"+uniqueid+"'><td><input type='checkbox' name='checkeditem[]'></td><td ><input type='hidden' value='"+routeid+"' name='stopover_startid[]'><input type='text' class='form-control' name='name1'size='4' value='"+startname+"' readonly /></td><td ><input type='hidden' value='"+endid+"' name='stopover_endid[]'><input type='text' class='form-control' name='name1'size='4' value='"+endname+"' readonly /></td><td><input type='text' class='form-control' name='stopover_price[]'size='4' value='0'  required /></td><td><input type='text' class='form-control' name='stopover_order[]'size='4' value='0'  required /></td></tr>";
-
-          var exists =  0;
-         $("table tbody").find("tr").each(function () {
-             var current_stock_id = $(this).attr('item-id');
-             if(current_stock_id == uniqueid)
-             {
-               exists = exists + 1;
-             }
-           });
-         if(exists == 0)
-         {
-                 $("#new-table tbody").append(markup);
-         }
-       });
-      // Find and remove selected table rows
-       $(".delete-row").click(function(){
-           $("table tbody").find('input[name="checkeditem[]"]').each(function(){
-               if($(this).is(":checked")){
-                   $(this).parents("tr").remove();
-               }
-           });
-       });
-       })
-</script>
+   <script type="text/javascript">
+       'use strict';
+
+        function generateRoutes(stations)
+        {
+            var routes = [];
+
+            var x;
+
+            for(x = 0; x<stations.length; x++) {
+                var y
+                for(y=(x+1); y<stations.length; y++) {
+                    var route = {
+                        "from": stations[x],
+                        "to": stations[y]
+                    };
+                    routes.push(route);
+                }
+            }
+
+            return routes;
+        }
+
+        $(document).ready(function() {
+            var stations = [];
+
+            $('select[name=station]').select2({
+                width: '100%',
+                allowClear: true,
+                minimumInputLength: 2,
+                placeholder: "Choose station...",
+                ajax: {
+                    type: 'GET',
+                    url: 'http://localhost:8000/transit/stations/suggest',
```

Comment:
I forgot to show you this bit.

You need to update this to a proper route.

I couldn't find where you had used ajax before.